e2e/harmony/load-bit.e2e.ts

```diff
@@ -0,0 +1,40 @@
+import { loadBit } from '@teambit/bit';
+import { ScopeAspect, ScopeMain } from '@teambit/scope';
+import { Workspace, WorkspaceAspect } from '@teambit/workspace';
+import { expect } from 'chai';
+import Helper from '../../src/e2e-helper/e2e-helper';
+import { HARMONY_FEATURE } from '../../src/api/consumer/lib/feature-toggle';
+
+describe.only('loadBit()', function () {
```

Comment:
@ranm8 , don't forget to remove the "only".

Reply:
thanks!